File: App/Backend/Backend/Backend/Models/UploadPostImage.cs
using Backend.Utils;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace Backend.Models
{
    public class UploadPostImage
    {
        public string ImageName { get; set; }
        public int PostId { get; set; }
        //[JsonConverter(typeof(Base64FileJsonConverter))]
        public IEnumerable<string> ImageData { get; set; }
    }
}

File: App/Frontend/frontendWeb/lib/models/uploadPostImage.dart
class UploadPostImage{
  String _imageName;
  int _postId;
  List<String> _imageData;

  UploadPostImage(this._postId,this._imageName, this._imageData);

  Map<String, dynamic> toJSON(){
    var map = Map<String, dynamic>();

    map["postId"] = _postId;
    map["imageName"] = _imageName;
    map["imageData"] = _imageData;

    return map;
  }
}